garak/_plugins.py

```diff
@@ -36,7 +36,9 @@ def default(self, obj):
 
 class PluginCache:
     _plugin_cache_file = _config.transient.basedir / "resources" / "plugin_cache.json"
-    _user_plugin_cache_file = _plugin_cache_file
+    _user_plugin_cache_file = (
+        _config.transient.cache_dir / "resources" / "plugin_cache.json"
+    )
```

Comment:
Another possible future enhancement, may consider adding a guard on json load of `plugin_cache.json` to fallback to system cache with a logged error when the local version is corrupted.